README.md

```diff
@@ -32,14 +32,14 @@ For Maven add to your `pom.xml`:
   <dependency>
     <groupId>io.opencensus</groupId>
     <artifactId>opencensus-api</artifactId>
-    <version>0.27.0</version>
+    <version>0.28.1</version>
```

Comment:
Out of curiosity: why did we skip 0.28.0?

Reply:
I did release 0.28.0, but only realised afterwards that I released my own fork's master, which didn't have the new changes 😓  So I did another 0.28.1 release with the new changes

Reply:
Makes sense, thanks!